sites/cheerpx/src/content/docs/12-reference/00-CheerpX-Linux-create.md

```diff
@@ -79,3 +79,173 @@ networkInterface?: NetworkInterface;
 This option configures network settings, which allows CheerpX to communicate over networks. For more detailed information about how CheerpX handles networking, including the use of Tailscale and overcoming browser limitations, see the [Networking](/docs/guides/Networking) guide.
 
 [Promise]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise
+
+### `authKey`
+
+```ts
+authKey?: string;
+```
+
+The `authKey` is an optional string used for authentication. It is usually taken from the URL, allowing the CheerpX Linux environment to be set up automatically without the user needing to enter it manually.
+
+**Example of usage:**
+
+```js
+let authKey = undefined;
+if (browser) {
+	let params = new URLSearchParams("?" + window.location.hash.substr(1));
+	authKey = params.get("authKey") || undefined;
+}
+const networkInterface = { authKey };
+```
+
+**Explanation**
+
+1. The `authKey` is taken from the URL, allowing the application to automatically use the authentication key provided by the browser.
+
+2. The `authKey` is added to the `networkInterface` object, ensuring the application can access the authentication data whenever needed in the CheerpX environment.
+
+### `controlUrl`
+
+```ts
+controlUrl?: string;
+```
+
+The `controlUrl` is an optional string parameter representing a URL used to manage or monitor the connection. It is typically fetched from the URL parameters and can be utilized for dashboard or configuration purposes.
+
+**Example of usage:**
+
+```js
+let controlUrl = undefined;
+if (browser) {
+	let params = new URLSearchParams("?" + window.location.hash.substr(1));
+	controlUrl = params.get("controlUrl") || undefined;
+}
+const dashboardUrl = controlUrl
+	? null
+	: "https://login.tailscale.com/admin/machines";
+const networkInterface = { controlUrl };
+```
+
+**Explanation**
+
+1. The `controlUrl` is taken from the URL and added to the `networkInterface` object, making it easy to connect to the appropriate control dashboard.
+
+2. If `controlUrl` isn’t provided, the app automatically uses a default URL (`dashboardUrl`) for monitoring or setup.
+
+### `loginUrlCb`
+
+```ts
+loginUrlCb?: (url: string) => void;
+```
+
+The `loginUrlCb` is a callback function designed to handle login URLs during the authentication process. It plays a critical role in the connection flow by setting the application's state to indicate readiness for login and resolving the URL required for the login process.
+
+**Example of usage:**
+
+```js
+function loginUrlCb(url) {
+	connectionState.set("LOGINREADY");
```

Comment:
The use of WebVM/Svelte specific code is not a good idea, since people will most likely not have this setup. It's not strictly required so it can be removed.